vhdlweb.wsgi

```diff
@@ -75,7 +92,7 @@ function compile() {
 startercode + \
 """</div>
 
-<script src="/ace-builds/src-noconflict/ace.js" type="text/javascript" charset="utf-8"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.4.3/ace.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
Oh, this is nice... didn't even know it existed!

Can you split this into a separate commit?  it's really separate from the save-button implementation, so it deserves its own commit/pull-request.